src/bin/rdcore/rootmap.rs

```diff
@@ -27,6 +27,10 @@ use libcoreinst::runcmd_output;
 use crate::cmdline::*;
 
 pub fn rootmap(config: &RootmapConfig) -> Result<()> {
+    // fail if we have more than 1 partition with boot label
+    if !check_single_partition("boot")? {
```

Comment:
Should this be a separate command instead? E.g. `rdcore verify-single-boot` or something.

The model for `rdcore` I think we should aim for is to try to keep commands modular/narrowly-scoped where fedora-coreos-config just "orchestrates" it. That way we can reorganize things on the FCOS side without having to cut another coreos-installer release/do backports. So e.g. for this, you could imagine we have more than one place where we want to check uniqueness of `boot`. Splitting it out of `rdcore rootmap` makes that possible.

Reply:
i was thikining abot this, so decided first to commit as it is and ask for upstream suggestions. will do